### ISSUE TITLE
Draper is incompatible with Ruby 3

### ISSUE THREAD

**Issue by brian-kephart** [drapergem/draper#884]
Similar to #869. My test suite spat this out:

```
gem_path/draper-4.0.1/lib/draper/helper_proxy.rb:32: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
gem_path/actionview-6.0.3/lib/action_view/helpers/translation_helper.rb:118: warning: The called method `l' is defined here
```

The referenced [method in ActionView](https://github.com/rails/rails/blob/d22216f262de92b1b659bc51ed3048447a0b36fd/actionview/lib/action_view/helpers/translation_helper.rb#L118) does indeed take keyword arguments. However, the [Draper method that causes the warning](https://github.com/drapergem/draper/blob/3e7faa4369efd1253a338369b4c8ad8e31ba8b68/lib/draper/helper_proxy.rb#L30-34) doesn't appear to be using keyword arguments:
https://github.com/drapergem/draper/blob/3e7faa4369efd1253a338369b4c8ad8e31ba8b68/lib/draper/helper_proxy.rb#L32

This appears to be a delegation issue as described [here](https://eregon.me/blog/2019/11/10/the-delegation-challenge-of-ruby27.html#the-delegation-challenge). I'm not well-up on delegation techniques or Draper usage, so I won't be submitting a PR for this myself. Hopefully someone else can shed some light on the issue.

**Comment by kulas115** [comment]
Seeing the same warning coming from `draper-4.0.1/lib/draper/automatic_delegation.rb:12`.

**Comment by zeeshangulzar** [comment]
I am getting same warning with following Tech Stack
**Ruby:** 2.7.1
**Rails** 6.0.3.3
**Draper:** 4.0.1 
```......./.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/draper-4.0.1/lib/draper/helper_proxy.rb:32: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
//.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/actionview-6.0.3.3/lib/action_view/helpers/translation_helper.rb:60: warning: The called method `t' is defined here```

**Comment by brian-kephart** [comment]
Updated the issue title. The deprecations in Ruby 2.7 weren't fixed, so Draper doesn't work with 3.0.

**Comment by tbuehlmann** [comment]
Addressed this in #894.

**Comment by brian-kephart** [comment]
Fixed in #885
